roles/container-engine/cri-o/defaults/main.yml

```diff
@@ -24,8 +24,9 @@ crio_stream_port: "10010"
 crio_required_version: "{{ kube_version | regex_replace('^v(?P<major>\\d+).(?P<minor>\\d+).(?P<patch>\\d+)$', '\\g<major>.\\g<minor>') }}"
 
 crio_kubernetes_version_matrix:
+  "1.19": "1.19"
   "1.18": "1.18"
   "1.17": "1.17"
   "1.16": "1.16"
```

Comment:
You can remove this line, 1.16 should not be needed anymore

Reply:
sorry missed that, will do, probably 1.17 also right?

Reply:
Yes not sure as we are in quite a weird state, still supporting 1.17 while at the same time not supporting it for future 2.15 :/

Reply:
(I would keep it for now)